sdk/core/Azure.Core/src/RequestMethod.cs

```diff
@@ -5,24 +5,57 @@
 
 namespace Azure.Core
 {
+    /// <summary>
+    /// Represents HTTP methods sent as part of <see cref="Request"/>
```

Comment:
nit:  "... as part of `a` Request"

nit:   Missing period at the end of the sentence.